TripItem new properties min and max persons in create and update forms

idalkilic opened the pull request.

rickvanderheijden committed:
Merge pull request #32 from rickvanderheijden/development

Development

dalkilici committed:
Merge branch 'development' of https://github.com/rickvanderheijden/TravelAgencyAngular into ismail

dalkilici committed:
List of trip items can be selected on create a trip

ArthurDoorgeest committed:
Fixed dropdown issue

dalkilici committed:
Merge branches 'development' and 'ismail' of https://github.com/rickvanderheijden/TravelAgencyAngular into ismail

dalkilici committed:
Trip-item create and update is updated with new properties min and max persons

ArthurDoorgeest reviewed (changes requested):
TripItems in trip gaat fout, even nakijken

dalkilici committed:
Trip-item create and update is updated with new properties min and max persons

dalkilici committed:
Merge branches 'development' and 'ismail' of https://github.com/rickvanderheijden/TravelAgencyAngular into ismail

dalkilici committed:
Pages en models updated, but still some problems in trip and user pages. User password is currently being sent as a string.  Must be solved

dalkilici committed:
Merge branches 'development' and 'ismail' of https://github.com/rickvanderheijden/TravelAgencyAngular into ismail

# Conflicts:
#	src/app/pages/trip-item/create/trip-item-create.component.html
#	src/app/pages/trip-item/update/trip-item-update.component.html
#	src/app/pages/user/user-create/user-create.component.ts
#	src/app/pages/user/user-update/user-update.component.ts
#	src/models/country.ts

ArthurDoorgeest committed:
fixed test that was failing

ArthurDoorgeest committed:
Merge branch 'development' into ismail

dalkilici committed:
Trip-items updated with new properties numberOfAttendees.
Trip updated with min and max persons, but destionation and tripitems selection still not done.

Merge branches 'development' and 'ismail' of https://github.com/rickvanderheijden/TravelAgencyAngular into ismail

# Conflicts:
#	src/app/pages/trip-item/create/trip-item-create.component.html
#	src/app/pages/trip-item/update/trip-item-update.component.html
#	src/app/pages/user/user-create/user-create.component.ts
#	src/app/pages/user/user-update/user-update.component.ts
#	src/models/country.ts
